ChangeLog

```diff
@@ -26,15 +26,25 @@ Changes
      Inserted as an enhancement for #1371
    * Add support for alternative CSR headers, as used by Microsoft and defined
      in RFC 7468. Found by Michael Ernst. Fixes #767.
-   * Correct many misspellings. Fixed by MisterDA #2371.
-   * Provide an abstraction of vsnprintf to allow alternative implementations
-     for platforms that don't provide it. Based on contributions by Joris Aerts
-     and Nathaniel Wesley Filardo.
-   * Fix clobber list in MIPS assembly for large integer multiplication.
-     Previously, this could lead to functionally incorrect assembly being
-     produced by some optimizing compilers, showing up as failures in
-     e.g. RSA or ECC signature operations. Reported in #1722, fix suggested
-     by Aurelien Jarno and submitted by Jeffrey Martin.
+   * Update the Mbed Crypto submodule to version 0.1.0b2.
+   * Add unit tests for AES-GCM when called through mbedtls_cipher_auth_xxx()
+     from the cipher abstraction layer. Fixes #2198.
+
+Features
+  * Add an experimental build option, USE_CRYPTO_SUBMODULE, to enable use of
```

Comment:
I think we should keep the 2.15.0 section for that. The 2.17.0 section is likely to have other things to say about it.